packages/beacon-node/src/api/rest/base.ts

```diff
@@ -84,34 +84,34 @@ export class RestApiServer {
     // Log all incoming request to debug (before parsing). TODO: Should we hook latter in the lifecycle? https://www.fastify.io/docs/latest/Lifecycle/
     // Note: Must be an async method so fastify can continue the release lifecycle. Otherwise we must call done() or the request stalls
     server.addHook("onRequest", async (req, _res) => {
-      const {operationId} = req.routeConfig as RouteConfig;
-      this.logger.debug(`Req ${req.id as string} ${req.ip} ${operationId}`);
+      const operationId = req.routeSchema.operationId as string;
+      this.logger.debug(`Req ${req.id} ${req.ip} ${operationId}`);
       metrics?.requests.inc({operationId});
     });
 
     server.addHook("preHandler", async (req, _res) => {
-      const {operationId} = req.routeConfig as RouteConfig;
-      this.logger.debug(`Exec ${req.id as string} ${req.ip} ${operationId}`);
+      const operationId = req.routeSchema.operationId as string;
+      this.logger.debug(`Exec ${req.id} ${req.ip} ${operationId}`);
     });
 
     // Log after response
     server.addHook("onResponse", async (req, res) => {
-      const {operationId} = req.routeConfig as RouteConfig;
-      this.logger.debug(`Res ${req.id as string} ${operationId} - ${res.raw.statusCode}`);
-      metrics?.responseTime.observe({operationId}, res.getResponseTime() / 1000);
+      const operationId = req.routeSchema.operationId as string;
+      this.logger.debug(`Res ${req.id} ${operationId} - ${res.raw.statusCode}`);
+      metrics?.responseTime.observe({operationId}, res.elapsedTime / 1000);
```

Comment:
As of latest release `getResponseTime()` has been deprecated (https://github.com/fastify/fastify/pull/5263) and will be removed in the next major release (v5)